uvm/hier/usr/lib/python3/dist-packages/tests/test_uvm.py

```diff
@@ -1210,6 +1217,16 @@ def test_300_feedback_link(self):
         match = re.search('^https://edge.arista.com/feedback$', feeddback_url)
         assert(match)
 
+    def test_302_cleanup_license_files(self):
+        """
+        Ensure that last 5 license files are present
+        """
+        # This should prune no of license files to 5
+        uvmContext.licenseManager().reloadLicenses(False)
+        license_files = find_files("/usr/share/untangle/conf/licenses/", "*licenses.js*")
+        # verify pruned no of license files to <=5
+        assert len(license_files) <= 5
```

Comment:
This test will fail on new systems.
I think if you call uvmContext.licenseManager().reloadLicenses(False) six times, that should populate to verify.

Reply:
@cblaise-untangle  added the code in loop and added skip tests check as testcase is taking around 15 seconds to complete.
![Screenshot from 2024-02-13 21-34-33](https://github.com/untangle/ngfw_src/assets/154513962/822f02b0-ac49-4638-9a42-eeb0f277c035)